pyproject.toml

```diff
@@ -47,7 +47,7 @@ optional = true
 [tool.poetry.group.docs.dependencies]
 Sphinx = { version = ">=8.2.3", python = ">=3.11" }
 sphinx-rtd-theme = { version = ">=3.0.2", python = ">=3.11" }
-tomli = ">=2.2.1"
+tomli = { version = ">=2.2.1", python = "<3.11" }
```

Comment:
The Sphinx version already requires Python >= 3.11, so we can just drop this dependency altogether.

Also, as the CI caught, we need to run `poetry lock` after doing this.

Reply:
indeed